HISTORY.md

```diff
@@ -0,0 +1,84 @@
+# Looking Back in Time at BackInTime
+
+*by Michael Büker, 2024*
+
+The history of the BackInTime project, which at the time of this writing already spans nearly 16 years, is best understood in four periods:
+
+1. The **First Era** from 2008 to 2012, releases 0.5 to ~1.0.12
+2. The **Second Era** from 2012 to 2019, releases ~1.0.14 to 1.2
+3. A **Dark Age** from 2019 to 2022, releases 1.2.0 to 1.3.2
+4. The **Third Era** since 2022, since release 1.3.3
+
+These periods correspond roughly to who was maintaining and developing BackInTime. Important technical and organizational changes happened at various moments in between.
+
+For details, refer to [CHANGES](CHANGES).
+
+## The First Era: 0.5 to ~1.0.12 (2008–2012)
+
+### Maintenance
+
+BackInTime was created by **Oprea Dan** and first published on a private blog in late 2008. Shortly thereafter, collaborative development started happening on Launchpad. Sometime around 2010, development and publication appears to have moved entirely to Launchpad, with the private blog being discontinued.
+
+### Core functionality
+
+At first, BackInTime used `diff` to compare the latest snapshot with the source, in order to check if a new snapshot was necessary. If the answer was yes, it would use `cp` to create a new snapshot.
```

Comment:
```suggestion
At first, _Back In Time_ used `diff` to compare the latest snapshot with the source, in order to check if a new snapshot was necessary. If the answer was yes, it would use `cp` to create a new snapshot.
```